backend/src/packages/auth/auth.controller.ts

```diff
@@ -149,6 +179,37 @@ class AuthController extends Controller {
       payload: options.user,
     };
   }
+
+  private async emailResetPasswordLink(
```

Comment:
```suggestion
  private async sendEmailResetPasswordLink(
```

Reply:
email here used as verb but I guess it can create confusion